### ISSUE TITLE
Make sure links in Header/Footer that are the same have the same HREF

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/vets-design-system-documentation#3175]
## Status
**[2024-08-08] [Fran]** This will be a DST ticket, since they are going to/are componentizing the header/footer. Will either transfer ticket, or share info so they can open their own ticket and then close this one.
**[2024-08-05] [Fran]** Pinged Randi and Chris in [Slack ](https://dsva.slack.com/archives/C52CL1PKQ/p1722895617411819)about async refining when they have a moment
**[2024-06-03]** Decided during prefinement meeting that refinement of this ticket will be async to attempt to provide estimate.
**[2024-05-30]** The Federal Header/Footer refresh is no longer happening; revisit this ticket to determine viability.
**[1/25/2024]** This will be work done after the 'new' Federal header/footer work is done, which is blocked on OFCIO research/disposition.

## Description
Automated accessibility tools flag issues where they find the same link text but different HREFs. This is a valid error but the tools take it very seriously. For example, they consider `https://www.va.gov/find-locations/` and `https://www.va.gov/find-locations` two different HREFs (because one doesn't have the end /). When we do the header/footer redesign, I want to be sure that links within the header and footer that are the same don't have this issue anymore just so I don't have to deal with these false positives.

I believe there are only 2 links this applies to:
- Contact us
- Find a VA Location

## Acceptance Criteria
- [ ] Ensure links that appear in both the header and footer have the exact same HREF
- [ ] Requires accessibility review

## Team
Please check the team(s) that will do this work.

- [ ] CMS Team
- [x] Public Websites
- [ ] Facilities
- [ ] Accelerated Publishing

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/3175?planning-poker) with Zenhub @Andrew565 @ataker @harshil1793 @it-harrison @jamigibbs @micahchiang @powellkerry @rmessina1010 @rsmithadhoc

**Comment by powellkerry** [comment]
The find a location link was updated in drupal/CMS to include the ending slash, the contact-us link in the header was updated in code to be an absolute link and match what was in the footer. Both links should be in sync now.
